src/test/java/com/exasol/adapter/dialects/postgresql/docgeneration/CapabilitiesReport.java

```diff
@@ -0,0 +1,95 @@
+package com.exasol.adapter.dialects.postgresql.docgeneration;
+
+import java.nio.file.Path;
+import java.util.Set;
+
+import com.exasol.adapter.capabilities.*;
+import com.exasol.adapter.dialects.SqlDialect;
+import com.exasol.adapter.dialects.SqlDialectFactory;
+import com.exasol.autogeneratedresourceverifier.AutogeneratedResource;
+
+import net.steppschuh.markdowngenerator.table.Table;
+
+public class CapabilitiesReport implements AutogeneratedResource {
```

Comment:
This looks like a generic class. Should it go to common-jdbc? We don't want to copy it for each dialect

Reply:
yes. But in a separate PR.

Reply:
I created an issue: https://github.com/exasol/postgresql-virtual-schema/issues/31